src/seedu/addressbook/Main.java

```diff
@@ -114,7 +114,10 @@ private CommandResult executeCommand(Command command)  {
         } catch (Exception e) {
             ui.showToUser(e.getMessage());
             throw new RuntimeException(e);
+        } catch (StorageOperationException s){
+            ui.showToUser(s.getMessage());
```

Comment:
As discussed in class, the control goes to showToUser method that prints a friendly error message. Is there something else we can do by handling exceptions more gracefully? Maybe save the program state to give the user a chance to change the file permission?